stan/math/opencl/kernel_generator/rowwise_reduction.hpp

```diff
@@ -16,6 +18,60 @@
 
 namespace stan {
 namespace math {
+namespace internal {
+
+template <typename Arg>
+struct matvec_mul_opt {
+  enum { is_possible = 0 };
```

Comment:
Though it's internal I think it would be good to doc these so it's easy to see what `is_possible` etc does from reading it here

Reply:
done